### PR TITLE
fix config injection of optional black/whitelists (second approach)

### PR DESCRIPTION
**Pull request by t1**
second approach to #74 

I've also added logback so I can see the log output.

### CODE REVIEW

**Comment by t1** [comment]
I didn't find an easier way. There seems to be some discussion going on for MP Config 2.0 (https://github.com/eclipse/microprofile-config/issues/446).

File: tck/pom.xml [hunk 1]
```diff
@@ -25,6 +25,12 @@
             <type>pom</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (phillip-kruger) [inline tck/pom.xml]: We use jboss logger. Does that not work for you ?

Comment (t1) [reply tck/pom.xml]: I see a warning in the logs that there is no slf4 binding

Comment (phillip-kruger) [reply tck/pom.xml]: mmm, wonder what is pulling in slf4j ? Maybe we can just exclude that dep  from whoever is pulling it in ? (less dependencies is better :) )

Comment (phillip-kruger) [reply tck/pom.xml]: seems like shrinkwrap-resolver-depchain

Comment (t1) [reply tck/pom.xml]: It's used by shrinkwrap and graphql-java... I assume it's easier to add the jboss-logger-bridge (I'm sure there is one).

Comment (t1) [reply tck/pom.xml]: `slf4j-jboss-logging`

Comment (t1) [reply tck/pom.xml]: fixed

Comment (phillip-kruger) [reply tck/pom.xml]: Cool !

**Comment by phillip-kruger** [comment]
Thanks !

**Pull request merged**